shell/assets/translations/en-us.yaml

```diff
@@ -1092,6 +1092,8 @@ catalog:
       label: Add new
       ariaLabel: Add a new repository
     appChartCard:
+      subHeaderItem:
+        missingVersionDate: Last updated date is not available for this chart
```

Comment:
I thought we were going to use the message: "No date available for this chart version"